src/core/CCOption.ml

```diff
@@ -111,6 +116,13 @@ let get_or ~default x =
   | None -> default
   | Some y -> y
 
+let apply_or f x =
```

Comment:
isn't that `fold`?

Reply:
It was motivated mostly by seeing places where CCString functions were being stringed together, combined with repeated CCOption's `get_or` to reset back to the original string if the string functions failed. Particularly the operator form would concisely shrink down that sort of pipeline to just the string functions.